### PR TITLE
fix: squad share OG and load

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Fix the OG texts
- Return if not available to adhere load of OG

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1142 #done

### CODE REVIEW

**Comment by idoshamun** [comment]
@rebelchris to make it work, we have to use ISR. Can we do it as part of this PR, please?

**Comment by rebelchris** [comment]
@idoshamun Added the ISR invocation :)

File: packages/webapp/pages/squads/[handle]/[token].tsx [hunk 1]
```diff
@@ -228,5 +239,5 @@ export function getStaticProps({
 }: GetStaticPropsContext<
   SquadReferralProps & ParsedUrlQuery
 >): GetStaticPropsResult<SquadReferralProps> {
-  return { props: { ...params } };
+  return { props: { ...params }, revalidate: 60 };
```
Comment (idoshamun) [inline packages/webapp/pages/squads/[handle]/[token].tsx:242]: What I originally meant is to have the query right here for all the information.
Many crawlers don't support javascript meaning any change we apply to OG during client rendering will not take affect.
Thanks to ISR, we can do it server-side and cache it.

**Review by idoshamun — APPROVED** [review]
Awesome!!! I love OpenGraph support 😬

**Pull request merged**